RickLeinecker/Fortuna
# index.js now checks if a user is logged in before rendering a page

**Description**
index.js now conditionally renders components based on whether a token is in cookies. Also, MainMenu is rendered instead of Login when a user is currently logged in.

We need to implement a API call that verify it is the logged in user's token, but currently it functions well.

Resolves #122 
Resolves #145 
Resolves #203 
Resolves #205 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Go to website.
2. Type in the URL to go to MainMenu.
3. Should lead back to Login.
4. Login.
5. Try to head back to Login through the URL.
6. Should lead back to MainMenu.

**Final Checklist:**
Go down the list and check them off.
